examples/predictors/lstm/xprize_predictor.py

```diff
@@ -84,10 +84,13 @@ def predict(self,
                 end_date_str: str,
                 npis_csv: str) -> pd.DataFrame:
         start_date = pd.to_datetime(start_date_str, format='%Y-%m-%d')
-        # end_date = pd.to_datetime(end_date_str, format='%Y-%m-%d')
+        end_date = pd.to_datetime(end_date_str, format='%Y-%m-%d')
 
         # Load the npis into a DataFrame, handling regions
         npis_df = self._load_original_data(npis_csv)
+        # TODO: Current logic forecasts for each day in this npis_df. Shrink it to the requested period until
```

Comment:
We now pass the list of IPs since inception (2020-01-01). But this code predicts for any date in the IP file. Temporarily fixing it by filtering out the passed IPs.
Will revisit when working on the validation stuff.